README.md

```diff
@@ -452,6 +454,14 @@ sudo ln -s /path/to/cuda /usr/local/cuda-X.X
 Please let us know on [the issue tracker](https://github.com/google/jax/issues)
 if you run into any errors or problems with the prebuilt wheels.
 
+### pip installation: Google Cloud TPU
+JAX also provides pre-built wheels for Google Cloud TPU. To install JAX along with appropriate
+versions of `jaxlib` and `libtpu`, you can run the following in your cloud TPU VM:
+```bash
+pip install --upgrade pip
+pip install --upgrade jax[tpu] -f https://storage.googleapis.com/jax-releases/jax_releases.html
```

Comment:
It's uglier now :(
```
pip install "jax[tpu]>=0.2.16" -f https://storage.googleapis.com/jax-releases/libtpu_releases.html
```
(The version specifier is needed because otherwise pip immediately gives up if there's an older version installed, since older versions don't have the [tpu] extra. We can eventually remove this once we don't expect versions without [tpu] to be present.)

Reply:
Done